$NetBSD: patch-src_Sigil_CMakeLists.txt,v 1.3 2013/10/20 20:25:23 wiz Exp $

FlightCrew pulls in pthread_cancel via
/usr/include/c++/bits/gthr-default.h
/usr/include/g++/bits/gthr.h
/usr/include/g++/ext/atomicity.h
/usr/include/g++/bits/basic_string.h
/usr/include/g++/string
/usr/include/g++/random
/usr/include/g++/bits/stl_algo.h
/usr/include/g++/algorithm
/usr/pkgsrc/editors/Sigil/work/.buildlink/include/boost/utility/swap.hpp
/usr/pkgsrc/editors/Sigil/work/.buildlink/include/boost/tuple/detail/tuple_basic.hpp
/usr/pkgsrc/editors/Sigil/work/.buildlink/include/boost/tuple/tuple.hpp
/usr/pkgsrc/editors/Sigil/work/src/FlightCrew/stdafx.h
/usr/pkgsrc/editors/Sigil/work/src/FlightCrew/Result.cpp
so link against libpthread.

--- src/Sigil/CMakeLists.txt.orig	2012-12-06 20:02:06.000000000 +0000
+++ src/Sigil/CMakeLists.txt
@@ -626,7 +626,7 @@ else()
     add_executable( ${PROJECT_NAME} WIN32 ${ALL_SOURCES} )
 endif()
 
-target_link_libraries( ${PROJECT_NAME} ${HUNSPELL_LIBRARIES} ${PCRE_LIBRARIES} tidyLib ${MINIZIP_LIBRARIES} ${BOOST_LIBS} ${XERCES_LIBRARIES} ${XERCESEXTENSIONS_LIBRARIES} ${FLIGHTCREW_LIBRARIES} ${QT_LIBRARIES} )
+target_link_libraries( ${PROJECT_NAME} ${HUNSPELL_LIBRARIES} ${PCRE_LIBRARIES} tidyLib ${MINIZIP_LIBRARIES} ${BOOST_LIBS} ${XERCES_LIBRARIES} ${XERCESEXTENSIONS_LIBRARIES} ${FLIGHTCREW_LIBRARIES} ${QT_LIBRARIES} -lpthread)
 
 #############################################################################
 
